src/modules/info/components/coordinateInfo/CoordinateInfo.js

```diff
@@ -0,0 +1,139 @@
+
+		this._mapService = mapService;
+		this._coordinateService = coordinateService;
+		this._translationService = translationService;
+		this._elevationService = elevationService;
+		this._shareService = shareService;
+	}
+
+	update(type, data, model) {
+		switch (type) {
+			case Update_Coordinate:
+				return { ...model, coordinate: data };
+			case Update_Elevation:
+				return { ...model, elevation: data };
+		}
+	}
+
+	/**
+	 * @override
+	 * @protected
+	 */
+	createView(model) {
+		const translate = (key) => this._translationService.translate(key);
+		const translateSilently = (key) => this._translationService.translate(key, [], true);
+		const { coordinate, elevation } = model;
+
+		if (coordinate) {
+			const coordinateRepresentations = this._mapService.getCoordinateRepresentations(coordinate);
+			const stringifiedCoords = coordinateRepresentations.map((cr) => {
+				const { label } = cr;
+				const stringifiedCoord = this._coordinateService.stringify(coordinate, cr);
+				const onClick = () => {
+					this._copyCoordinateToClipboard(stringifiedCoord);
+				};
+				return html`
+					<span class="label">${translateSilently(label)}</span><span class="coordinate">${stringifiedCoord}</span>
+					<span class="icon">
+						<ba-icon
+							class="close"
+							.icon="${clipboardIcon}"
+							.title=${translate('info_coordinateInfo_copy_icon')}
+							.size=${1.5}
+							@click=${onClick}
+						></ba-icon>
+					</span>
+				`;
+			});
+
+			return html`
+				<style>
+					${css}
+				</style>
+
+				<div class="container">
+					<ul class="content selectable">
+						${stringifiedCoords.map((strCoord) => html`<li class="r_coordinate">${strCoord}</li>`)}
+						${elevation
+							? html`<li class="r_elevation">
+									<span class="label">${translate('info_coordinateInfo_elevation_label')}</span><span class="coordinate">${elevation}</span>
+								</li>`
+							: nothing}
+					</ul>
+				</div>
+			`;
+		}
+		return nothing;
+	}
+
+	async _copyCoordinateToClipboard(stringifiedCoord) {
+		try {
+			await this._shareService.copyToClipboard(stringifiedCoord);
+			emitNotification(`"${stringifiedCoord}" ${this._translationService.translate('info_coordinateInfo_clipboard_success')}`, LevelTypes.INFO);
+		} catch {
+			const message = this._translationService.translate('info_coordinateInfo_clipboard_error');
+			emitNotification(message, LevelTypes.WARN);
+			console.warn('Clipboard API not available');
+		}
+	}
+
+	async _getElevation(coordinate) {
+		try {
+			const elevation = await this._elevationService.getElevation(coordinate);
+			this.signal(Update_Elevation, elevation);
+		} catch (e) {
+			this.signal(Update_Elevation, null);
+			throw e;
+		}
+	}
+
+	static get tag() {
+		return 'ba-coordinate-info';
+	}
+
+	/**
+	 * @property {module:domain/coordinateTypeDef~Coordinate} coordinate - the coordinate to display
+	 */
```

Comment:
According to our technical doc guide, the property should be documented in the header of the class